custom_components/ramses_cc/__init__.py

```diff
@@ -27,15 +26,17 @@
 from homeassistant.helpers.typing import ConfigType
 from homeassistant.helpers.update_coordinator import DataUpdateCoordinator
 
-from .const import BROKER, DOMAIN, SIGNAL_UPDATE
-from .coordinator import RamsesBroker
-from .schemas import (
-    SCH_DOMAIN_CONFIG,
+from .const import (
+    BROKER,
+    CONF_ADVANCED_FEATURES,
+    CONF_MESSAGE_EVENTS,
```

Comment:
So, my practice is for constants that are strings to start with `SZ_`.

HA does use `CONF_` for strings used in config (and voluptuous schemas), or `ATTR_`, etc.

So when should it be `CONF_*` (or `ATTR_`), and when it be `SZ_*`?

You seem to have it about right.